tests/cases/standalone/common/range/interval.result

```diff
@@ -20,25 +20,18 @@ Affected Rows: 8
 
 SELECT ts, host, min(val) RANGE (INTERVAL '1 year') FROM host ALIGN (INTERVAL '1 year') ORDER BY host, ts;
 
-+---------------------+-------+----------------------------------------------------------------------------+
-| ts                  | host  | MIN(host.val) RANGE IntervalMonthDayNano("950737950171172051122527404032") |
-+---------------------+-------+----------------------------------------------------------------------------+
-| 1970-01-01T00:00:00 | host1 | 0                                                                          |
-| 1970-12-27T00:00:00 | host1 | 2                                                                          |
-| 1970-01-01T00:00:00 | host2 | 4                                                                          |
-| 1970-12-27T00:00:00 | host2 | 6                                                                          |
-+---------------------+-------+----------------------------------------------------------------------------+
-
-SELECT ts, host, min(val) RANGE (INTERVAL '1' year) FROM host ALIGN (INTERVAL '1' year) ORDER BY host, ts;
-
-+---------------------+-------+----------------------------------------------------------------------------+
-| ts                  | host  | MIN(host.val) RANGE IntervalMonthDayNano("950737950171172051122527404032") |
-+---------------------+-------+----------------------------------------------------------------------------+
-| 1970-01-01T00:00:00 | host1 | 0                                                                          |
-| 1970-12-27T00:00:00 | host1 | 2                                                                          |
-| 1970-01-01T00:00:00 | host2 | 4                                                                          |
-| 1970-12-27T00:00:00 | host2 | 6                                                                          |
-+---------------------+-------+----------------------------------------------------------------------------+
+Error: 3000(PlanQuery), DataFusion error: Error during planning: Year or month interval is not allowed in range query: IntervalMonthDayNano("950737950171172051122527404032")
```

Comment:
Why does it fail?

Reply:
We don't allow using year or month in range query now as how many days in a month or a year is unspecific. The behavior of the range query with `month or year interval` before this PR is incorrect. Users still can use `30 day` or `365 day` if they want to treat a month as 30 days and a year as 365 days.